internal/cli/atlas_backups_checkpoints_list.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type atlasBackupsCheckpointsListOpts struct {
+	*globalOpts
+	clusterName  string
+	pageNum      int
+	itemsPerPage int
+	store        store.CheckpointsLister
+}
+
+func (opts *atlasBackupsCheckpointsListOpts) init() error {
+	if opts.ProjectID() == "" {
+		return errMissingProjectID
+	}
+
+	if opts.clusterName == "" {
```

Comment:
you don't need this, please do 
```go
_ = cmd.MarkFlagRequired(flags. ClusterName)
```
in line 90, the only reason we do this check for the project id is that a flag is not the only way to set it